docs/integration-handbook.md

```diff
@@ -0,0 +1,246 @@
+# Autonomous System Integration Handbook
+This page describes how to integrate your autonomous system (AS) to the Formula Student Driverless Simulator (FSDS).
+The rules and procedures set out in this document will be used during the FSOnline competition.
+
+## High-level overview
+Your AS is expected to continuously run a ROS master.
+The simulator will launch a node ([fsds_ros_bridge](ros-bridge.md)) that connects to your ROS system.
+This node will publish sensor data and listen for vehicle setpoints.
+When the simulation is done, the node is closed and your AS will no longer be able to interact with the simulator.
+[A more in-depth explanation of the FSDS system can be found here.](system-overview.md)
+
+Integrating your autonomous system with a simulator is a matter of subscribing and publishing to topics and acting accordingly.
+
+*During testing, your AS and the ROS bridge/simulator will probably run on the same machine (your local computer most likely). However, during the competition, the AS and the ROS bridge/simulator will be on different machines (cloud servers) and communicate over a local network.*
```

Comment:
before merging, check this @SijmenHuizenga !

Reply:
Checked, looks good!